core/dispatcher/src/main/scala/whisk/core/invoker/Invoker.scala

```diff
@@ -369,15 +368,15 @@ class Invoker(
      * it is omitted entirely.
      *
      * @param strings The list of strings
-     * @param sizeLimit The limit of strings to get
+     * @param limit The limit of strings to get
      */
-    def truncateLogs(strings: Vector[String], sizeLimit: ObjectSize): Vector[String] = {
+    def truncateLogs(strings: Vector[String], limit: LogLimit): Vector[String] = {
         val stringStream = strings.toStream
         val cummulativeSizes = stringStream.scanLeft(0.bytes) { (acc, current) => acc + current.sizeInBytes }.tail
-        val truncatedLogs = stringStream.zip(cummulativeSizes).takeWhile(_._2 < sizeLimit).map(_._1).toVector
+        val truncatedLogs = stringStream.zip(cummulativeSizes).takeWhile(_._2 < limit.megabytes.megabytes).map(_._1).toVector
```

Comment:
can you add an apply method to that limit.megabytes() eschews the double megabytes.megabytes?